docs/source/user-guide/configs.md

```diff
@@ -23,7 +23,6 @@ Comet provides the following configuration settings.
 
 | Config | Description | Default Value |
 |--------|-------------|---------------|
-| spark.comet.ansi.enabled | Comet does not respect ANSI mode in most cases and by default will not accelerate queries when ansi mode is enabled. Enable this setting to test Comet's experimental support for ANSI mode. This should not be used in production. | false |
```

Comment:
This highlights that our configs vary depending on the version we build against and we currently only publish one version of our configs.

Ideally we should generate configs per spark version and publish all of them.

Reply:
I will file an issue for this

Reply:
Thanks, I plan to associate this config with `spark.sql.ansi.enabled`. So when Spark enables ANSI, so does Comet